content/en/_index.html

```diff
@@ -183,37 +183,39 @@ <h2 class="section-label">Resources</h2>
-    youtube="nGLpUCPX8JE"
-    title="The Evolution of Flux v2 with Stefan Prodan" %}}
-  Stefan introduces Flux v2, explains why it was reshaped into a composable continuous delivery solution that goes beyond Git sync to accommodate multi-tenancy, infrastructure dependencies and cluster-api fleet management.
+    url="https://youtube.com/playlist?list=PL9lTuCFNLaD3fI_g-NXWVxopnJ0adn65d"
+    thumbnail="0v5bjysXTL8"
+    title="The Power of GitOps with Flux Playlist" %}}
+  This playlist is designed to show the powerful GitOps capabilities of Flux (and the GitOps Toolkit). Videos include project overviews, reviews, technical demos, and walk-throughs of our guides.
   {{% /blocks/resource %}}
 
   {{% blocks/resource
-    youtube="0v5bjysXTL8"
-    title="The Power of GitOps with Flux v2 & GitOps Toolkit with Leigh Capili" %}}
-  Leigh demos bootstrapping with GitOps Toolkit, app deployment, and monitoring with Prometheus.
+    youtube="Fzzg4KQavEU"
+    title="Flux Deep Dive with Stefan Prodan" %}}
+  Stefan walks through what Flux is, who it’s for and its components. Then gives a demo on how to use Flux for delivering infrastructure and applications changes onto Kubernetes.
   {{% /blocks/resource %}}
 
   {{% blocks/resource
-    youtube="JcKUawSQfQ0"
-    title="Flux v2 for Helm Users with Scott Rigby" %}}
-  Scott covers what to keep in mind as Helm 2 support ends, the benefits of Helm Controller, and how to migrate from Helm Operator to Helm Controller.
+    youtube="NLU6b82FhFs"
+    title="From Wild West to Flux Multi-tenancy with Mae Large & Russ Parmer" %}}
+  Mae and Russ share how the State Farm GitOps team helped to simplify application deployment while meeting compliance requirements, enable the admin team to handle scale and embrace the declarative nature of Kubernetes.
   {{% /blocks/resource %}}
 
   {{% blocks/resource
-    youtube="7W27tAv7Tvs"
-    title="Managing Remote Clusters with Flux v2 (CAPI demo) with Leigh Capili" %}}
-  Leigh covers cluster API integration, dependency management & ordering, security model updates, and delegation & multi-tenancy via users.
+    youtube="gfyo6VqgD_k"
+    title="Flux's Catalog - Keeping Track of Things! with Leigh Capili" %}}
+  Leigh Capili gives us a black belt, under-the-covers look at how Flux's kustomize-controller tracks the child objects for a Kustomization in the Kubernetes API.
   {{% /blocks/resource %}}
 
   {{% blocks/resource
-    youtube="R6OeIgb7lUI"
-    title="Flux v2 overview, demo, & review with Viktor Farcic" %}}
-  Stefan introduces Flux v2, explains why it was reshaped into a composable continuous delivery solution that goes beyond Git sync to accommodate multi-tenancy, infrastructure dependencies and cluster-api fleet management.
+    youtube="laMwuG8r7Tw"
+    title="Handling Dependencies with Flux with Jason Morgan" %}}
+  Kubernetes provides a simple way to apply configuration against a cluster but sometimes the order of operations matters.
```

Comment:
Jason walks through how to use Flux’s dependency manager to ensure that your apps get added to your mesh every time.